MapboxNavigation/NavigationView.swift

```diff
@@ -75,6 +75,34 @@ open class NavigationView: UIView {
     lazy var muteButton = FloatingButton.rounded(image: Images.volumeUp, selectedImage: Images.volumeOff)
     lazy var reportButton = FloatingButton.rounded(image: Images.feedback)
     
+    public enum FloatingButtonsPositions {
```

Comment:
For consistency with the map SDK’s `MGLOrnamentPosition` enumeration, consider calling this enumeration `MapOrnamentPosition` (singular). We’ll probably eventually need to use this same enumeration for other kinds of controls that are positioned over the map. For example, eventually, we may add the ability to place the floating buttons along the bottom or place the speed limit view on the same side as the floating buttons.

Reply:
Thanks for the suggestion, I'll update the name and add it to the jazzy